apps/browser/qml/pages/components/HistoryItem.qml

```diff
@@ -67,5 +76,5 @@ ListItem {
     }
 
     ListView.onRemove: animateRemoval()
-    onClicked: view.load(model.url, model.title)
+    onClicked: { console.log("XXXXXXXXXXx loading: " + model.url); view.load(model.url, model.title) }
```

Comment:
not sure if it's something that I broke, or whether it's broken on devel currently, but attempting to open a history item from the popup menu (rather than the url autocompletion list) currently fails.

Reply:
works on latest devel, so I must have broken it somehow.  curious.